Fix:

warning: variable 'doc' is uninitialized when used within its own initialization [-Wuninitialized]

https://sourceforge.net/p/aewan/bugs/14/
--- vlayer.c
+++ vlayer.c
@@ -9,7 +9,7 @@ static void _get_cell_LYR(Cell *r_cell, int x, int y, void *ud) {
 
 /* callback that returns a cell from a document composite */
 static void _get_cell_COMPOSITE(Cell *r_cell, int x, int y, void *ud) {
-   Document *doc = (Document*) doc;
+   Document *doc = (Document*) NULL;
    *r_cell = document_calc_effective_cell(doc, x, y);
 }
 
